.env-sample

```diff
@@ -5,6 +5,7 @@ SERVER_PORT=8090
 LOG_LEVEL=debug
 
 CORE_DB_CONNECTION_STRING=
```

Comment:
lets add a quick comment explaining what the connection string should be and why we have a separate variable for the database name just like in the readme